tests/pom.xml

```diff
@@ -148,25 +148,6 @@
                     </exclusions>
                 </dependency>
             </dependencies>
-            <build>
-                <plugins>
-                    <plugin>
-                        <groupId>org.codehaus.mojo</groupId>
-                        <artifactId>build-helper-maven-plugin</artifactId>
-                        <executions>
-                            <execution>
-                                <id>add-321cdh-test-src</id>
-                                <goals><goal>add-test-source</goal></goals>
-                                <configuration>
-                                    <sources>
-                                        <source>${project.basedir}/src/test/320+/scala</source>
```

Comment:
perhaps we should move this directory to be 320+-noncdh to match our source code

Reply:
How about 320+-noncdh-nondb as it's not testing db either

Reply:
thatsfine